ArduCopter/Attitude.cpp

```diff
@@ -130,9 +130,11 @@ float Copter::get_surface_tracking_climb_rate(int16_t target_rate, float current
     }
 
     static uint32_t last_call_ms = 0;
+    const float current_alt = inertial_nav.get_altitude();
+    const float current_alt_target = pos_control->get_alt_target();
+    const float dt = G_Dt;
```

Comment:
This is only used once - just drop G_Dt in?

Reply:
sounds good, replaced dt with G_Dt